### PR TITLE
fix some lint issues:

### PR DESCRIPTION
**Pull request by andrewsmedina**
* remove a shadow declaration
* check that err are not nil in tests

### CODE REVIEW

File: adapters/postgres/postgres_test.go [hunk 1]
```diff
@@ -172,11 +175,15 @@ func TestJoinByRequest(t *testing.T) {
 	})
 	Convey("Join missing param", t, func() {
 		r, err := http.NewRequest("GET", "/prest/public/test?_join=inner:test2:test2.name:$eq", nil)
+		So(err, ShouldNotBeNil)
```
Comment (felipeweb) [inline adapters/postgres/postgres_test.go]: err Should be nil here

File: adapters/postgres/postgres_test.go [hunk 2]
```diff
@@ -172,11 +175,15 @@ func TestJoinByRequest(t *testing.T) {
...
+
 		_, err = JoinByRequest(r)
 		So(err, ShouldNotBeNil)
 	})
 	Convey("Join invalid operator", t, func() {
 		r, err := http.NewRequest("GET", "/prest/public/test?_join=inner:test2:test2.name:notexist:test.name", nil)
+		So(err, ShouldBeNil)
```
Comment (felipeweb) [inline adapters/postgres/postgres_test.go]: err Should be nil here

**Comment by andrewsmedina** [comment]
the build is ok now :)

**Comment by avelino** [comment]
thank you @andrewsmedina

**Pull request merged**